core/encoding/uuid/reading.odin

```diff
@@ -170,19 +186,46 @@ time_v6 :: proc "contextless" (id: Identifier) -> (timestamp: u64) {
 }
 
 /*
-Get the timestamp of a version 7 UUID.
+Get the timestamp of a version 6 UUID.
+
+Inputs:
+- id: The identifier.
+
+Returns:
+- timestamp: The timestamp, in 100-nanosecond intervals since 1582-10-15.
+*/
+time_v6 :: proc "contextless" (id: Identifier) -> (timestamp: time.Time) {
+	delta := cast(time.Duration)(raw_time_v6(id) - HNS_INTERVALS_BETWEEN_GREG_AND_UNIX) * 100
+	return time.time_add({}, delta)
```

Comment:
This should probably just be a straight up `Time{_nsec=...}` because subtraction notwithstanding, we're converting a moment between epochs, not calculating an interval.